drag-and-drop/library/src/utils/runtime-flow-state.ts

```diff
@@ -484,6 +502,8 @@ export class RuntimeFlowState {
 
     this.realAssetMappings.delete(assetSymbol);
     parentDropTarget.value = this.computeViewForDropTarget(parentDropTarget);
+
+    this.handleDndStateChange(this.exportContent());
```

Comment:
To make it a bit cleaner, Instead of calling the passed in function directly here with arguments, there should be a local function that does any of the necessary pre-processing and then called the passe in function.